Python/Algorithms/data_structures/list_structs/fifo_lifo.py

```diff
@@ -0,0 +1,87 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Abstract_FIFO_LIFO:
+    
+    @abstractmethod
+    def insert(self,value):
+        return
+    
+    @abstractmethod
+    def remove(self,value):
+        return
+
+class FIFO_LIFO_Common(metaclass=ABCMeta):
```

Comment:
This is also for Any linked list. It is important to know future implications before naming is concerned. This is not just for FIFO and LIFO, this is for any SinglyLL